docs/conf.py

```diff
@@ -37,18 +39,10 @@
 # ones.
 extensions = [
     "sphinx.ext.intersphinx",
+    "sphinxcontrib.jquery",
     "pydoctor.sphinx_ext.build_apidocs",
 ]
 
-try:
-    import rst2pdf.pdfbuilder
-
-    extensions.append("rst2pdf.pdfbuilder")
```

Comment:
I left the comment about this inside the PR description.

This was a drive-by cleanup.

> As a drive-by, I have removed the Gitter embeeded chat box, as it's no longer available.
I have also removed the PDF code, since we don't build the PDF version of the docs.


![](https://deviq.com/static/bdfd476a88cf0804fb5a999a487253af/7f61c/BoyScoutRule-400x400.webp)